airbyte-webapp/src/locales/en.json

```diff
@@ -486,6 +486,7 @@
   "tables.connectionDeleteConfirm": "Confirm connection deletion",
   "tables.sourceDeleteModalText": "Deleting a source cannot be undone without a full re-sync. Note that:\n  - All connections with this source will be deleted, including their past logs and configurations.\n  - No existing data in the destination will be altered",
   "tables.destinationDeleteModalText": "Deleting a destination cannot be undone. Note that:\n  - All connections with this destination will be deleted, including their past logs and configurations.\n  - No existing data in the destination will be altered.",
```

Comment:
One more thing 😅 : I noticed that while just deleting sources or destinations, the message is now confusing because no connections will be deleted if they don't appear. We can simplify this text to:

```
"Deleting a source cannot be undone without a full re-sync. No existing data in the destination will be altered."

"Deleting a destination cannot be undone. No existing data in the destination will be altered."
```

This way the user only worries about connections being deleted when there are any.